app/controllers/concerns/api/articles_controller.rb

```diff
@@ -118,6 +123,21 @@ def unpublish
       end
     end
 
+    def search
+      # I temporarily added a new search endpoint in the interest of getting the chatGPT plugin live without changing
+      # the existing index endpoint. There are some experiments which we want to conduct which I think makes sense on
+      # a new endpoint rather than an existing one. We may want to refactor the index one in the future.
+      @articles = Articles::ApiSearchQuery.call(params)
+
+      # This adds some inconsistency where we omit the body markdown when the response has more than 1 article because
+      # ChatGPT cannot process the long body request.
```

Comment:
What if we just omit body_markdown all the time? What do we achieve from providing body_markdown only when there's one article?

Reply:
It gives ChatGPT the ability to summarize a article.

Reply:
I suspect this falls into the camp of something we can transition from the somewhat naive current implementation to something a little bit more sophisticated after we validate the MVP.

Reply:
💯